src/com/company/Main.java

```diff
@@ -6,9 +6,59 @@
 
 public class Main {
 
-    public static void main(String[] args) {
+    public static Address createAddress(final Scanner scanner) {
+        System.out.println("Enter the address");
+        System.out.println("City:");
+        final String cityNewBank = scanner.next();
+        System.out.println("Zip code:");
+        final int zipCodeNewBank = scanner.nextInt();
+        System.out.println("Street:");
+        final String streetNewBank = scanner.nextLine();
+        System.out.println("House number:");
+        final int houseNumberNewBank = scanner.nextInt();
+        System.out.println("Apartment number:");
+        final int apartmentNumberNewBank = scanner.nextInt();
+        return new Address(cityNewBank, zipCodeNewBank, streetNewBank, houseNumberNewBank, apartmentNumberNewBank);
+    }
+
+    public static void createBank(final Scanner scanner, final List<Bank> banks) {
```

Comment:
Wouldn't it be better to make this method similar to `createAddress`, that is returning the created object, instead of putting it into banks list?